node/deepgram-transcribe-video/README.md

```diff
@@ -0,0 +1,69 @@
+# 📝 Deepgram Transcribe Video
+
+A Node Cloud Function for transcribing a video using the [DEEPGRAM API](https://developers.deepgram.com/).
+
+_Example input 1:_
+
+```json
+{
+    "payload": "{\"fileUrl\": \"https://rawcdn.githack.com/deepgram-devs/transcribe-videos/62fc7769d6e2bf38e420ee5224060922af4546f7/deepgram.mp4\"}"
+}
+```
+
+_Example output 1:_
+
+```json
+{
+    "success": true,
+    "deepgramData": {"A long": "Object"},
+}
+```
+
+_Example input 2:_
+
+```json
+{
+     "payload": "{}"
+}
+```
+
+_Example output 2:_
+
+```json
+{
+    "success": false,
+    "message": "Please provide a file url in payload"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Let's remove references to environment since that's from the older images:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```